components/CardView.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-card
+    max-width="550"
+    max-height="350"
+    class="d-flex align-content-start grey lighten-5"
+  >
+    <div align="center" justify="center">
+      <v-img
+        class="rounded-circle"
+        width="100"
+        height="100"
+        src="https://raw.githubusercontent.com/github/explore/80688e429a7d4ef2fca1e82350fe8e3517d3494d/topics/vue/vue.png"
+      ></v-img>

```

Comment:
https://vuejs.org/v2/style-guide/#Component-name-casing-in-templates-strongly-recommended